x/incentives/simulation/genesis.go

```diff
@@ -14,23 +15,29 @@ import (
 
 // Simulation parameter constants
 const (
-	ParamsDistrEpochIdentifier       = "distr_epoch_identifier"
+	ParamsDistrEpochIdentifier = "distr_epoch_identifier"
 )
 
 // RandomizedGenState generates a random GenesisState for gov
 func RandomizedGenState(simState *module.SimulationState) {
-      // Parameter for how often rewards get distributed
+	// Parameter for how often rewards get distributed
 	var distrEpochIdentifier string
 	simState.AppParams.GetOrGenerate(
 		simState.Cdc, ParamsDistrEpochIdentifier, &distrEpochIdentifier, simState.Rand,
 		func(r *rand.Rand) { distrEpochIdentifier = GenParamsDistrEpochIdentifier(r) },
 	)
 
 	incentivesGenesis := types.GenesisState{
-		Params: types.Params {
+		Params: types.Params{
 			DistrEpochIdentifier: distrEpochIdentifier,
 		},
 		// Gauges: gauges,
+		LockableDurations: []time.Duration{
```

Comment:
TODO: Double check these make sense with epoch times and the defaults in pool-incentives